airbyte-webapp/src/components/JobItem/JobItem.tsx

```diff
@@ -97,7 +102,13 @@ const JobItem: React.FC<IProps> = ({ shortInfo, ...props }) => {
           >
             {isOpen ? (
               isJobEntity(props) ? (
-                <JobLogs id={jobMeta.id} jobIsFailed={isFailed} />
+                <>
+                  <ErrorDetails
+                    attempts={isJobEntity(props) ? props.job.attempts : undefined}
```

Comment:
nit: looks like this whole `<ErrorDetails` component is already within an `isJobEntity(props)`

```suggestion
                    attempts={props.job.attempts}
```